### PR TITLE
LVRUBY-50: Add views counter to news model

### PR DESCRIPTION
**Pull request by kmandziy**
## JIRA ticket

https://ssu-jira.softserveinc.com/browse/LVRUBY-50

## Code reviewers

- [ ] @okohuch 
- [ ] @nataliakozoriz 

## Summary of issue

Add views counter to news model

## Summary of change

Added field views in News table

## Review checklist

- [ ] Rubocop doesn't raise any warnings and offenses
- [ ] New features are covered with new specs
- [ ] All specs pass

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -48,6 +49,8 @@
     t.string "last_name", null: false
     t.text "about_me"
     t.datetime "remember_created_at"
+    t.string "provider"
```
Comment (okohuch) [inline db/schema.rb]: these two are not related to your changes. remove them

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

File: db/schema.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (kmandziy) [reply db/schema.rb]: Changed

Comment (kmandziy) [reply db/schema.rb]: Removed

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.65%** covered at **2.8** hits/line
